src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -5,22 +5,22 @@
       "Size": 3568
     },
     "assemblies/FormsViewGroup.dll": {
-      "Size": 7251
+      "Size": 7240
     },
     "assemblies/Java.Interop.dll": {
-      "Size": 61695
+      "Size": 61902
     },
     "assemblies/Microsoft.Win32.Primitives.dll": {
-      "Size": 3834
+      "Size": 3833
     },
     "assemblies/Mono.Android.dll": {
-      "Size": 437730
+      "Size": 442644
```

Comment:
I think maybe we were near the limit here, and this change pushed it over the edge?

Main is currently passing, and it looks like we don't keep the build artifact of this file for passing tests.